### DISCUSSION TITLE
bibmanager alfred integration

### DISCUSSION THREAD

**Discussion by AaronDavidSchneider** [pcubillos/bibmanager#97] [Show and tell]
Hi folks,

for those of you using OSX: I wrote a small [alfred](https://www.alfredapp.com) workflow to enable a `bibm browse` like behavior in my alfred search! 

Check it out:
https://github.com/AaronDavidSchneider/alfred-bibmanager

Cheers
Aaron

**Comment by pcubillos** [comment]
This is great! Thank you!
